### ISSUE TITLE
Tag substitution does not work across imported definitions

### ISSUE THREAD

**Issue by phackstock** [IAMconsortium/nomenclature#385]
As an example, common-definitions defines the tag `Level-1 Species`. 
This currently cannot be used in a local workflow repository that imports common-definitions.
Not sure if this would be needed often though.

**Comment by danielhuppmann** [comment]
Interesting new feature idea, but could cause issues if tags are defined in the parent and child repo.

**Comment by phackstock** [comment]
@danielhuppmann, good point, I'm definitely not sure for the moment if we even want this feature.
